std/algorithm/setops.d

```diff
@@ -949,7 +949,8 @@ See also: $(LREF multiwayMerge)
 auto multiwayUnion(alias less = "a < b", RangeOfRanges)(RangeOfRanges ror)
 {
     import std.algorithm.iteration : uniq;
-    return ror.multiwayMerge.uniq;
+    import std.functional : not;
+    return multiwayMerge!less(ror).uniq!(not!less);
```

Comment:
nitpick: please don't mix non-UFCS and UFCS.

Either `ror.multiwayMerge!less.uniq!(not!less)` or `uniq!(not!less)(multiwayMerge!less(ror))`.